my-code/movie-search/src/Containers/MovieCatalogue/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
```

Comment:
You are importing a very big library to use just one function (`map`).
Besides, map is a ES6 function that you can use without lodash.

Honestly, most of lodash's functions can be easily dropped in favour of ES6

Reply:
Oh, that makes sense. I thought lodash was necessary for this case. I'll refactor this then :)